src/components/AddCommentCard.jsx

```diff
@@ -8,31 +44,67 @@ export default function AddCommentCard() {
               <div className="card-header comment-card-header">
                 <h5 className="card-title">Add Comment</h5>
               </div>
-              <div className="mb-3">
-                <label htmlFor="exampleFormControlInput1" className="form-label">
-                  Username:
-                </label>
-                <input
-                  type="email"
-                  className="form-control"
-                  id="exampleFormControlInput1"
-                  placeholder="name@example.com"
-                />
-              </div>
-              <div className="mb-3">
-                <label htmlFor="exampleFormControlTextarea1" className="form-label">
-                  Comment:
-                </label>
-                <textarea
-                  className="form-control"
-                  id="exampleFormControlTextarea1"
-                  rows="3"
-                ></textarea>
-              </div>
+              <form action="submit" onSubmit={handleSubmit}>
+                <div className="mb-3">
+                  <label htmlFor="username" className="form-label">
+                    Username:
+                  </label>
+                  <select
+                    className="form-select"
+                    aria-label="Default select example"
+                    id="username"
+                    placeholder="Enter your username"
+                    required
+                    onChange={handleChange}
+                    onBlur={(event) => {
+                      const error =
+                        document.getElementById("username_error_msg");
+                      if (event.target.value.length < 3) {
+                        error.innerText = "Please enter a username";
+                      } else {
+                        error.innerText = "";
```

Comment:
Rather than having to declare error here then mess with these properties, you could have some error state which would clean things up a little. You can then have the ternary to return whichever html you want, depending on whether the error state is true or false 👍